### PR TITLE
Apply machine scope to options that specify paths

### PR DESCRIPTION
**Pull request by rchande**

### CODE REVIEW

**Comment by rchande** [comment]
Addresses https://github.com/OmniSharp/omnisharp-vscode/issues/3116

File: package.json [hunk 1]
```diff
@@ -486,12 +487,12 @@
                   "items": {
                     "type": "string"
                   },
-                  "description": "Array of symbol server URLs (example: http\u200b://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
+                  "description": "Array of symbol server URLs (example: http​://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
```
Comment (gregg-miskelly) [inline package.json]: nit: this got reformatted to removed the escape characters. Can you add them back? Run `npm run gulp generateOptionsSchema`

File: package.json [hunk 2]
```diff
@@ -404,6 +404,7 @@
           "type": "object",
           "description": "Options to use with the debugger when launching for unit test debugging. Any launch.json option is valid here.",
           "default": {},
+          "scope": "machine",
```
Comment (gregg-miskelly) [inline package.json]: The machine scoping is because of `sourceFileMap`?

Comment (rchande) [reply package.json]: That was my thinking, but I think I agree it's worth roaming the other parts. Will revert.

File: package.json (hunk 1 above)
Comment (rchande) [reply package.json]: Done.

**Pull request merged**